_Week5/group4/demonstration.do

```diff
@@ -0,0 +1,12 @@
+global username "/Users/geenapanzitta/Library/CloudStorage/Box-Box/"
+
+cd "${username}ECON490/Week5/"

```

Comment:
You may be missing a forward slash after {username}. Please correct me if I'm mistaken, which is certainly possible.

Reply:
I think that beyond that, the file path wasn't change once moved to git. Thanks for bringing that to our attention!